File: add/add.go
package main
import ( "fmt" )

func main() {
	var X int
	var Y int
	fmt.Println("add of X AND Y")
	fmt.Print("Enter X: ")
    _, err := fmt.Scan(&X)
    if err != nil {
        fmt.Println("Error reading X:", err)
        return
    }

    // Asking for input of Y
    fmt.Print("Enter Y: ")
    _, err = fmt.Scan(&Y)
    if err != nil {
        fmt.Println("Error reading Y:", err)
        return
    }

    // Calculating and displaying the result
    fmt.Printf("X + Y = %d + %d = %d\n", X, Y, X+Y)

}

File: add/add.py

def main():
    print("add of X AND Y")
    try:
        X = int(input("Enter X: "))
        Y = int(input("Enter Y: "))
    except ValueError as e:
        print(f"Error reading input: {e}")
        return

    # Calculating and displaying the result
    print(f"X + Y = {X} + {Y} = {X + Y}")

if __name__ == "__main__":
    main()